tutorials/intellijDebugger.md

```diff
@@ -6,7 +6,140 @@ pageNav: 3
 
 # {{ title }}
 
+Before diving into tracing code with IntelliJ IDEA’s debugger, it’s essential to understand key debugger features and their purpose.
```

Comment:
This sentence is not relevant to this page, as the reader of this page doesn't necessarily come from the AB3 code  tracing tutorial.